specification/components/r4/schemas/responses/R4-NotFound.yaml

```diff
@@ -0,0 +1,20 @@
+description: |
+  Where status code 404 (Not Found) is returned then an [NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) will be included in the body, as detailed below.
+  Check diagnostics property for specific information regarding the error.
+  
+  | issue.details.coding.code | issue.code | Description                                                                                                                    |
+  | ------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------ |
+  | REC_NOT_FOUND             | not-found  | The origin server did not find a current representation for the target resource or is not willing to disclose that one exists. |
```

Comment:
I think this description could be a bit more specific, perhaps "The supplied URL or Resource does not exist on the server"?

Reply:
Changed to 

> The supplied URL or Resource does not exist on the server